x-pack/plugins/data_enhanced/public/search/ui/connected_search_session_indicator/connected_search_session_indicator.tsx

```diff
@@ -196,6 +201,9 @@ export const createConnectedSearchSessionIndicator = ({
           viewSearchSessionsLink={searchSessionsManagementUrl}
           searchSessionName={searchSessionName}
           saveSearchSessionNameFn={saveSearchSessionNameFn}
+          startedTime={startTime}
```

Comment:
// nit 

maybe pass the whole meta object in?

Reply:
Not sure if this is a matter of preference or if there are real benefits, but I prefer to keep props as flat as possible.

one of the examples: easier to track excessive re-renders than when using deeply nested objects
another: Adding a new prop to the service level `meta` interface would extend the component interface but it is possible that the new prop is not needed on the component. So I'd have to create a new similar interface for a component to keep it right.